src/encoded/project/authentication.py

```diff
@@ -8,7 +8,7 @@ def login(self, context, request, *, samesite):
 
     def namespaced_authentication_policy_authenticated_userid(self, namespaced_authentication_policy, request,
                                                               set_user_info_property):
-        set_user_info_property = False
+        set_user_info_property = True
```

Comment:
I still haven't figured out what this does. (i.e.,  I think a comment wouldn't hurt)